webapp/channels/src/components/admin_console/data_grid/data_grid_row.tsx

```diff
@@ -14,43 +14,41 @@ type Props = {
     row: Row;
 }
 
-class DataGridRow extends React.Component<Props> {
-    renderCell(row: Row, column: Column) {
-        const style: CSSProperties = {};
-        if (column.width) {
-            style.flexGrow = column.width;
-        }
-
-        if (column.textAlign) {
-            style.textAlign = column.textAlign;
-        }
-
-        if (column.overflow) {
-            style.overflow = column.overflow;
-        }
-
-        return (
-            <div
-                key={column.field}
-                className={classNames('DataGrid_cell', column.className)}
-                style={style}
-            >
-                {row.cells[column.field]}
-            </div>
-        );
+const renderCell = (row: Row, column: Column) => {
```

Comment:
This can be considered a component itself, so let's change this to something like `DataGridCell` with props `row` and `column`.

We could also consider memoizing the style, but the calculation is so simple and the potential re-render on the child component is so small, that probably is better not to memoize.

Reply:
Applied the changes. Note that `dataGridCell` is not capitalised  (as opposed to DataGridRow) due to the naming rules.